scripts/project.js

```diff
@@ -24,6 +31,9 @@ Project.loadAll = function(inputData) {
   });
 };
 
+//TODO: Refactor the below code to obtain only the head when getting the eTag.
+
+//CHECKS IF THERE IS NO CODEPROJECTS @ LOCALSTORAGE. IF NONE, LOADS IT FROM SERVER(codeProjects.json), STRINGIFIES IT AND STORES IT IN LOCALSTORE. THEN, GRABS eTag, STRINGIFIES IT, STORES IN LOCALSTORAGE. THEN CALLS ITSELF.
```

Comment:
Awesome job, capturing the eTag from the header metadata and using it to load only what you need to.